src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -550,6 +561,55 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			// is this request the full path to a real file?
+			if (File.Exists(localPath) && Path.IsPathRooted(localPath))
+			{
+
+				// currently this only applies to languageDisplay.css, settingsCollectionStyles.css, and customCollectionStyles.css
+				var cssFile = Path.GetFileName(localPath);
+				if ((cssFile == "languageDisplay.css") || (cssFile == "settingsCollectionStyles.css") || (cssFile == "customCollectionStyles.css"))
```

Comment:
Seems this path will never be taken, because other changes ensure that these files are put in the HTML using unrooted paths, in fact, the unadorned filenames? Or maybe that's only for edit pages and this is needed in other cases?